Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -14159,6 +14261,21 @@ end
 					},
 					Notes = "Returns the current OS time, as a unix time stamp (number of seconds since Jan 1, 1970)",
 				},
+				IsBiomeCold =
+				{
+					Params =
+					{
+						{
+							Name = "Biome",
+							Type = "EMCSBiome",
+						},
+					},
+					Returns =
+					{
+						{ Type = "boolean" },
+					},
+					Notes = "Returns true if the biome is cold. Doesn't report very cold biomes, use IsBiomeVeryCold() for those.",
```

Comment:
How about using the doxy-comments from the actual C++ code?
```cpp
/** Returns true if the biome is very cold
(has snow on ground everywhere, turns top water to ice, has snowfall instead of rain everywhere).
Doesn't report mildly cold biomes (where it snows above certain elevation), use IsBiomeCold() for those. */
extern bool IsBiomeVeryCold(EMCSBiome a_Biome);

/** Returns true if the biome is cold
(has snow and snowfall at higher elevations but not at regular heights).
Doesn't report Very Cold biomes, use IsBiomeVeryCold() for those. */
extern bool IsBiomeCold(EMCSBiome a_Biome);
```